wayland-client/src/calloop.rs

```diff
@@ -0,0 +1,223 @@
+//! Utilities for using an [`EventQueue`] from wayland-client with an event loop that performs polling with
+//! [`calloop`](https://crates.io/crates/calloop).
+
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![warn(missing_docs, missing_debug_implementations)]
+#![forbid(unsafe_op_in_unsafe_fn)]
+#![allow(clippy::new_without_default)]
```

Comment:
Please don't add such attributes in the module like this, they are either redundant with the crate-level ones or have no reason to be here.

Reply:
Ah, sorry I must have copied the file from the wrong subdirectory. I think this is a leftover from the attempted separate `WaylandSource` crate. I'll remove it!